pkg/crcontrollers/drive/drivecontroller.go

```diff
@@ -33,6 +33,12 @@ type Controller struct {
 	log            *logrus.Entry
 }
 
+const (
+	ignore uint8 = 0
```

Comment:
Is that types of standard return codes from Reconcile?

Reply:
No, these constants just to make code more readable and update/delete drive CR in one place.

Reply:
Got it, thanks